test/functional/ui/screen.lua

```diff
@@ -1263,7 +1297,7 @@ function Screen:print_snapshot(attrs, ignore)
                .."{\n"..table.concat(attrstrs, "\n").."\n}")
   end
   print( "\nscreen:expect"..(keys and "{grid=" or "(").."[[")
```

Comment:
side-note: @bfredl any objection to removing `keys` boolean (and associated logic), so that it always produces the kwargs form? That would simplify this logic a bit.

Reply:
Let's do it.